### ISSUE TITLE
[Bug] [Android] StackLayout receives accessibility focus on Android API 21-22

### ISSUE THREAD

**Issue by rachelkang** [dotnet/maui#1089]
### Description

StackLayout receives accessibility focus in Android API 21-22. (This is not an issue in API 23 and later versions. And this is not an issue on native Android for any version.) When TalkBack is activated, the StackLayout appears to receive focus. In doing so, all the labels in the layout are read aloud at once, and the labels are not individually accessible. 

The same issue is seen in XF/XCT. Setting `AutomationProperties.IsInAccessibleTree="False"` on the layout does not help. Android Layout Inspector does not work on Android emulators/devices < API 23, so debugging is also a bit of a challenge.

### Steps to Reproduce

1. Run MAUI on Android device/emulator using API 21-22
2. Navigate to SemanticsPage
3. Activate TalkBack
4. Focus will likely start on the first entry (which is also weird behavior) - swipe left until focus moves to the StackLayout

### Expected Behavior
StackLayout should not receive screen reader focus. Individual labels within the layout should receive focus.

### Actual Behavior
StackLayout receives screen reader focus. Individual labels within the layout do not receive focus.

### Basic Information
- Platform Target Frameworks: 
  - Android: API 21 and API 22
- Affected Devices: Pixel emulators

### Screenshots

<img width="1792" alt="Screenshot 2021-05-26 at 10 43 48 AM" src="https://user-images.githubusercontent.com/21988533/119695264-ffcdbd80-be1b-11eb-8a3d-4baf9684ef49.png">

### Workaround
None found.

**Comment by PureWeen** [comment]
Possibly related https://github.com/dotnet/maui/issues/11742

**Comment by Zhanglirong-Winnie** [comment]
Verified this issue with Visual Studio Enterprise 17.6.0 Preview 2.0. Not repro on android physical device (API 31) with a MAUI template project.
[MauiApp4.zip](https://github.com/dotnet/maui/files/11197586/MauiApp4.zip)
